File: Modules/HumanResource/PayrollSettings/EmployerContributions/EmployerContributionsForm.cs
using Serenity.ComponentModel;
using System;

namespace HRMSoftware.PayrollSettings.Forms;

[FormScript("PayrollSettings.EmployerContributions")]
[BasedOnRow(typeof(EmployerContributionsRow), CheckNames = true)]
public class EmployerContributionsForm
{
    public string Description { get; set; }
    public double Amount { get; set; }
}

File: Modules/ServerTypes/PayrollSettings/EmployerContributionsForm.ts
import { StringEditor, DecimalEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface EmployerContributionsForm {
    Description: StringEditor;
    Amount: DecimalEditor;
}

export class EmployerContributionsForm extends PrefixedContext {
    static formKey = 'PayrollSettings.EmployerContributions';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!EmployerContributionsForm.init)  {
            EmployerContributionsForm.init = true;

            var w0 = StringEditor;
            var w1 = DecimalEditor;

            initFormType(EmployerContributionsForm, [
                'Description', w0,
                'Amount', w1
            ]);
        }
    }
}